src/main.ts

```diff
@@ -78,17 +78,18 @@ async function runForReleaseEvent(vt: VirusTotal) {
         await vt.monitorItems(await github.downloadReleaseAsset(octokit, asset, path.join(context.tmpDir(), asset.name)), inputs.monitorPath).then(upload => {
           outputAnalysis.push(`${asset.name}=${upload.url}`);
           core.info(`${asset.name} successfully uploaded. Check detection analysis at ${upload.url}`);
-          release.body = release.body.concat(`\n* [\`${asset.name}\`](${upload.url})`);
+          release.body = release.body.concat(`\n  * [\`${asset.name}\`](${upload.url})`);
         });
       } else {
         await vt.files(await github.downloadReleaseAsset(octokit, asset, path.join(context.tmpDir(), asset.name))).then(upload => {
           outputAnalysis.push(`${asset.name}=${upload.url}`);
           core.info(`${asset.name} successfully uploaded. Check detection analysis at ${upload.url}`);
-          release.body = release.body.concat(`\n* [\`${asset.name}\`](${upload.url})`);
+          release.body = release.body.concat(`\n  * [\`${asset.name}\`](${upload.url})`);
         });
       }
     });
   });
+  release.body = release.body.concat(`\n\ncreated by [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal)</details>\n`);
```

Comment:
I prefer to have the link as it was before. See prev comment.

Reply:
See comment above.